src/user-guide/sharing-workflows/sharing-workflows.rst

```diff
@@ -0,0 +1,285 @@
+greatest possible.
+
+However, any negations are applied with priority.
+
+Note, default in site config does not contribute to permissions, this is used
+in the case where a user does not appear in the `c.CylcUIServer.user-authorization`.
+
+.. note::
+
+   Spellings of UIServer configurations should be strictly adhered to. Using,
+   for example, `c.CylcUIServer.user-authorisation` rather than
+   `c.CylcUIServer.user-authorization` would result in cylc silently ignoring
+   this configuration.
+
+Methods of Assigning Permissions
+--------------------------------
+Assigning permissions can be done in two ways:
+
+ - using individual Cylc operations, e.g. ``play``, ``pause``
+ - using predefined access groups: ``READ``, ``CONTROL``, ``ALL``.
+
+Using both methods is supported, e.g ["READ", "stop", "pause"]
+
+
+Individual Operations
+^^^^^^^^^^^^^^^^^^^^^ 
+To assign users permissions, you can list the operations you wish to grant
+
+.. code-block:: python
+
+  c.CylcUIServer.user_authorization = {
+      "user1": ["read", "pause", "play"]
+      }
+
+Provided you have permission to give `user1` these permissions, as defined in the
+site configuration file, this will result in `user1` being able to see your
+workflows (from the `read` operation), and `pause` and `play` your workflows.
+
+
+Removing permissions can be achieved by prepending the operation with a ``!``.
+For example,
+
+.. code-block:: python
+
+   c.CylcUIServer.user_authorization = {
+       "group:groupA": ["read", "play", "stop"],
+       "user2": ["!stop"]
+   }
+
+Again, providing your site configuration permits you to grant this access,
+this configuration, for `user2` (who is a member of system group `groupA`) would
+result in them having "read" and "play" access. The the access to stop your workflows
+has been removed and this action will be forbidden.
+
+Access Groups
+^^^^^^^^^^^^^
+For convenience, cylc operations available in the ui have been bundled into
+access groups. These should be capitalized to distinguish from cylc operations.
+
+We currently support ``READ``, ``CONTROL`` and ``ALL`` and to remove permissions
+to operations in these groups, use ``!READ``, ``!CONTROL``, ``!ALL``.
+
+
+.. csv-table:: Access Group Mappings
+   :header: "Operation", "READ", "CONTROL", "ALL"
+
+   "Broadcast", , , "X"
+   "Ext-trigger",, "X", "X"
+   "Hold",, "X", "X"
+   "Kill",, "X", "X"
+   "Message",, "X", "X"
+   "Pause",, "X", "X"
+   "Ping","X", , "X"
+   "Play",, "X", "X"
+   "Poll",, "X", "X"
+   "Read","X", , "X"
+   "Release",, "X", "X"
+   "ReleaseHoldPoint",, "X", "X"
+   "Reload",, "X", "X"
+   "Remove",, "X", "X"
+   "Resume",, "X", "X"
+   "SetGraphWindowExtent",, "X", "X"
+   "SetHoldPoint",, "X", "X"
+   "SetOutputs",, "X", "X"
+   "SetVerbosity",, "X", "X"
+   "Stop",, "X", "X"
+   "Trigger",, "X", "X"
+
+
+.. note::
+
+   ``READ`` access group (which exands to ``["read", "ping"]``) is a method for
+   granting non-interactive access.
```

Comment:
?

Reply:
As in, with READ, you are providing view-only type access to other users. They will be unable to interact with workflows. 
I can remove this if you think it is obvious enough, through the naming of the access group.

Reply:
It's the non-interactive access bit that's confusing me.